opentelemetry/src/sdk/metrics/aggregators/ddsketch.rs

```diff
@@ -49,32 +49,32 @@ pub fn ddsketch(config: &DDSketchConfig, kind: NumberKind) -> DDSKetchAggregator
 ///
 /// In order to support both negative and positive inputs, DDSketchAggregator has two DDSketch store within itself to store the negative and positive inputs.
 #[derive(Debug)]
-pub struct DDSKetchAggregator {
+pub struct DdSKetchAggregator {
```

Comment:
Should the "K" be lowercase here?

Reply:
:+1: Good catch. Wonder why clippy didn't find it